### ISSUE TITLE
Doesn't work!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#431]
```
What's the problem?

Despite getting the confirmation pop-up that my post had been published, it 
never appeared at Blogger.



What browser are you using?

Chrome v 14.0.835.186



What version of ScribeFire are you running?

1.9

```

-----
Original issue reported on code.google.com by `gsshar...@gmail.com` on 24 Sep 2011 at 10:46

**Comment by GoogleCodeExporter** [comment]
```
What is your blog's URL? Did the post eventually appear?
```

-----
Original comment by `cfinke` on 13 Oct 2011 at 8:40
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
The blog's URL is http://twirlinginthelight.blogspot.com/

No, the post never appeared.
```

-----
Original comment by `gsshar...@gmail.com` on 13 Oct 2011 at 9:44

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 4 Nov 2011 at 2:15
* Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in Scribefire version 4.2 for Chrome
```

-----
Original comment by `scribefi...@gmail.com` on 3 Sep 2013 at 9:18
* Changed state: **Fixed**
